src/presentation/http/router/note.test.ts

```diff
@@ -415,8 +300,89 @@ describe('Note API', () => {
     });
 
     test.todo('Returns 400 when parentId has incorrect characters and length');
-    test.todo('Return 200 when user is not the creator, but a team member with a Write role');
-    test.todo('Return 403 when user has no Write role and he is not a creator');
+
+    test('Returns 200 when user is a team member with a Write role', async () => {
+      /** Create test user */
+      const creator = await global.db.insertUser();
+
+      /** Create another test user */
+      const teamMember = await global.db.insertUser();
+
+      /** Create test note */
+      const note = await global.db.insertNote({
+        publicId: 'juiKow8wlZ',
```

Comment:
PublicId is generated automatically starting with this pull request https://github.com/codex-team/notes.api/pull/196
Remove the redundancy.